tests/test_randomizer.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+import dice
+import randomizer
+
+
+def test_add_ability_to_pool(ability_d4):
+    randomizer.add_ability_to_pool(ability_d4)
+    assert len(randomizer.Pool) == 1
+
+
+def test_remove_ability_from_pool(ability_d4):
+    initial_len = len(randomizer.Pool)
+
+    randomizer.remove_ability_from_pool(ability_d4)
+    assert len(randomizer.Pool) < initial_len
+
+
+def test_empty_pool(ability_d4, ability_d8):
+    randomizer.add_ability_to_pool(ability_d4)
+    randomizer.add_ability_to_pool(ability_d8)
+
+    randomizer.empty_pool()
+    assert len(randomizer.Pool) == 0
+
+
+def test_pop_value_from_pool(ability_d8):
+    randomizer.add_ability_to_pool(ability_d8)
+
+    pop_value = randomizer.pop_value_from_pool()
```

Comment:
add and pop could've been moved to other fixtures, so your test will contain only assert statements. I prefer it this way, because it makes me think 'what state do I want to test?' first and then 'how can I achieve this state?'. This way I move all the setup code into fixtures, allowing me in tests to focus only on their purpose: testing the state, nothing else.